MAKE-website/scripts/page_quiz_info.js

```diff
@@ -67,11 +107,21 @@ const quizzes = [
 
 function renderQuizInfo() {
     const quiz_container = document.getElementById("quiz-list-container");
+    const general_quiz_container = document.getElementById("general-quiz-container");
 
     let divs = [];
 
     for (let quiz of quizzes) {
-        divs.push(generateQuizDiv(quiz));
+        let quiz_div = generateQuizDiv(quiz);
```

Comment:
def change this to a find function for the general quiz, then append the rest checking to see if it's general and skipping if it is

Reply:
Does this commit 8f3b29b011041601feae96e5af7b17148ac06527 have the changes you had in mind?

Reply:
yup!